CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ TensorFlow coding style.
     [tensorflow/core](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/core)
     and
     [tensorflow/python](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python).
-    TensorFlow has reached version 1 and hence cannot make
+    TensorFlow has reached version 2 and hence cannot make
```

Comment:
I think "passed version 1.0" is better as that is the cutting point (according to [semver](https://semver.org/#spec-item-5))) after which backward-compatibility must be maintained.

Reply:
🙌 @mihaimaruseac Done